CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+# More details are here: https://help.github.com/articles/about-codeowners/
+
+# Default owners for everything in the repository
+*   @recognizegroup/team-technology @jvanenckevort @godzillar
```

Comment:
Hij geeft aan dat deze fout is.

Reply:
Ik denk dat 'ie nu gefixt is, expliciet team-technology toegevoegd.

Reply:
> @jvanenckevort we hebben het gisteren gehad over het automatisch doorzetten van een release naar een PR via actions. Dat mis ik hierin.

This has not yet been developed, within the last paragraph, I shortly described this automatic process, which will be described in detail when it has been built.

Reply:
Dat snap ik, maar wellicht goed om even uit te schrijven en te checken met @jeroenvleeming en @godzillar , dat was het idee, toch?

Reply:
This will be described in a ticket first and documentation text will be added once finished.